CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+v1.10.0-RC1 / 2020-05-13
```

Comment:
Once RC candidates are done and a final release is made, can we also have all the notes under a major release like `v1.10.0`

Reply:
yes.

Reply:
This PR was generated by script so it is for RC1 only. I thought of creating a separate PR using script for RC1, RC2 and v1.10.0.

Reply:
Done. Changelog for RC2 and main release in the same PR now.